user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIDataSource.md

```diff
@@ -31,3 +31,37 @@ The GQI will request data.
 #### Returns
 
 A [GQIPage](xref:GQI_GQIPage) with the data.
+
+> [!TIP]
+> In many cases, not all data is immediately necessary for generating a query result. Thus, it's beneficial to partition rows into separate pages. This allows for fetching and processing individual pages as needed, enhancing both speed and memory efficiency. For example:
+
+#### Examples
+
+##### Paged data retrieval
+
+The following example illustrates how the `HasNextPage` boolean can be used to indicate that there are additional pages. If `HasNextPage` is set to `true`, GQI may call `GetNextPage` again when more data is needed.
```

Comment:
Are we referring here to the variable or the property?
In any case, 'boolean' should be changed to either 'variable' or 'property'